__init__.py

```diff
@@ -8,7 +8,7 @@
 from .utils.logger import logger
 
 from typing import Any, Optional
-from PyQt5.QtWidgets import QWidget
+from aqt.qt import QWidget
```

Comment:
It seems like we can ensure compatibility by using aqt.qt instead of PyQt6

Reply:
Yes, given that we fit with the new naming conventions of Qt6.